src/main/java/org/broadinstitute/hellbender/cmdline/ReadFilterArgumentDefinitions.java

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.hellbender.cmdline;
+
+/**
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ */
+public final class ReadFilterArgumentDefinitions {
+    private ReadFilterArgumentDefinitions(){}
```

Comment:
Maybe some `StandardArgumentDefinitions` constants related with `ReadFilter` should live here too:
* READ_FILTER_LONG_NAME
* DISABLE_READ_FILTER_LONG_NAME
* DISABLE_TOOL_DEFAULT_READ_FILTERS
* READ_FILTER_SHORT_NAME
* DISABLE_READ_FILTER_SHORT_NAME

Reply:
@droazen - should I move this too?

Reply:
Let's get tests passing on this branch before moving any more constants around, as this is a somewhat error-prone process.

Reply:
Good point!